core/js/test/JsConverterTest.kt

```diff
@@ -22,7 +22,7 @@ class JsConverterTest {
     @Test
     fun toInstantTest() {
         val kotlinReleaseEpochMilliseconds = 1455494400000
-        val releaseDate = Date(milliseconds = kotlinReleaseEpochMilliseconds)
+        val releaseDate = Date(milliseconds = kotlinReleaseEpochMilliseconds.toDouble())
```

Comment:
This is likely no longer needed, since we reverted the approach with expect Date

Reply:
Fixed